### PR TITLE
forum.eset.com forums.laptopvideo2go.com

### PR DESCRIPTION
**Pull request by FadeMind**
https://github.com/darkreader/darkreader/issues/5503

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -3967,6 +3982,34 @@ a[data-mentionid],
...
+forums.laptopvideo2go.com
+
+INVERT
+.ipsReact_button
+
+CSS
+.ipsBadge {
+    --darkreader-bg--badge--background: var(--darkreader-neutral-background) !important;
+}
+.ipsReact_reactCount > a,
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:3994]: These 2 selectors can be added to the list of selectors under this line.

**Review by Gusted — CHANGES_REQUESTED** [review]

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (FadeMind) [reply src/config/dynamic-theme-fixes.config:3994]: @Gusted nope . ipsBadge need to be as is.

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:3994]: I meant
```CSS
.ipsReact_reactCount > a,
.ipsReact_reactCount > span {
    background-color: var(--darkreader-neutral-background) !important;
}
```

As the one at line 4008 is the same declaration.

Comment (FadeMind) [reply src/config/dynamic-theme-fixes.config:3994]: This applied in latest commit fix.

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -3967,6 +3981,33 @@ a[data-mentionid],
 
 ================================
 
+forums.laptopvideo2go.com
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:3984]: I just noticed they share the same selectors, so might be worth removing this one and add
`forums.laptopvideo2go.com`
to Line 3985, like:
```css
forum.eset.com
forums.laptopvideo2go.com
```

**Comment by FadeMind** [comment]
@Gusted  changes applied.

**Review by Gusted — APPROVED** [review]
Looks good! Thank you.

**Pull request merged**